v2.0/releases/index.md

```diff
@@ -8,8 +8,20 @@ Use the version selector at the top-right of this page to view a different relea
 
 ## v2.0.0
 
+Notable Features:
+
+- Most of Calico's components have been rewritten from python to golang, including:
+  calicoctl, libcalico (as libcalico-go), calico-cni, libnetwork-plugin, most of felix, and parts of calico/node.
+- calicoctl introduces a new
+  [resource-based UX model]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands).
+- Felix's golang rewrite adds major performance improvements particularly around resync / catch-up behavior.
```

Comment:
Other advantages of the port:
- improved packaging that now vendors our dependencies; should avoid Python dependency issues such as incompatible urllib3/requests versions
- now shares a lot more code via libcalico-go
- introduction of a dataplane driver API, which will allow for non iptables dataplanes in future